baselines/commoncontrols.md

```diff
@@ -909,13 +909,13 @@ Google Takeout services SHALL be disabled.
 
 #### GWS.COMMONCONTROLS.12.1v0.3 Instructions
 1.  Sign in to https://admin.google.com as an administrator.
-2.  Select **Account** then **Google Takeout.**
+2.  Select **Data** -\> **Data Import & Export** -\> **Google Takeout**.
 3.  Select **User access to Takeout for Google services**.
-4.  For services without an individual admin control, select **Services without an individual admin control** then **Edit.**
+4.  For services without an individual admin control, select **Services without an individual admin control** then **Edit**.
 5.  Select **Don't allow for everyone**.
 6.  Click **Save**.
 7.  For services with an individual admin control, under **apps** select the checkbox next to **Service name** and select **Don't allow**.
-8.  Click **Save.**
+8.  Click **Save**.
```

Comment:
There actually isn't a save button after the last step.  The admin center immediately applies the change when you click "Don't allow"
```suggestion
8.  Click **Save**.
```

Reply:
I think this might be an artifact from when there was a save button, because I think the admin console as a whole auto saves when you exit so if we change it here, we might want to change it all over.

Reply:
I agree that we'd want to be consistent. But I just double checked, at least within Common Controls, all the other instances of "Save" are still needed.

Reply:
Ok sounds good, let's be consistent with what is in the admin console so let's remove it.